include/argparse/argparse.hpp

```diff
@@ -1104,6 +1104,21 @@ class ArgumentParser {
     return *this;
   }
 
+  explicit operator bool() const {
+    auto arg_used = std::any_of(m_argument_map.cbegin(),
+                                m_argument_map.cend(),
+                                [](auto &it) {
+                                    return it.second->m_is_used;
+                                });
+    auto subparser_used = std::any_of(m_subparser_used.cbegin(),
+                                      m_subparser_used.cend(),
+                                      [](auto &it) {
+                                          return it.second;
+                                      });
+
+    return m_is_parsed && (arg_used || subparser_used);
```

Comment:
Why don't we use just `m_is_parsed`? I have a subcommand without arguments and `operator bool()` returns false for it.